ytmusicapi/navigation.py

```diff
@@ -78,6 +78,19 @@
 CAROUSEL_TITLE = ["header", "musicCarouselShelfBasicHeaderRenderer", *TITLE]
 CARD_SHELF_TITLE = ["header", "musicCardShelfHeaderBasicRenderer", *TITLE_TEXT]
 FRAMEWORK_MUTATIONS = ["frameworkUpdates", "entityBatchUpdate", "mutations"]
+ACCOUNT_INFO = [
+    "actions",
+    0,
+    "openPopupAction",
+    "popup",
+    "multiPageMenuRenderer",
+    "header",
+    "activeAccountHeaderRenderer",
+]
+ACCOUNT_RUNS_TEXT = ["runs", 0, "text"]
+ACCOUNT_NAME = [*ACCOUNT_INFO, "accountName", *ACCOUNT_RUNS_TEXT]
+ACCOUNT_CHANNEL_HANDLE = [*ACCOUNT_INFO, "channelHandle", *ACCOUNT_RUNS_TEXT]
+ACCOUNT_PHOTO_URL = [*ACCOUNT_INFO, "accountPhoto", "thumbnails", 0, "url"]
```

Comment:
I'd prefer to put these constants in the `parse_account_info` function for now, they're unlikely to be used anywhere else

Reply:
I moved them in to `get_account_info` since I took your suggestions and moved the code out of the library parser